src/dotnet/APIView/APIView/Languages/CodeFileBuilder.cs

```diff
@@ -102,7 +102,7 @@ public CodeFile Build(IAssemblySymbol assemblySymbol, bool runAnalysis)
 
             var node = new CodeFile()
             {
-                Name = assemblySymbol.Name,
+                Name = $"{assemblySymbol.Name} ({assemblySymbol.Identity.Version})",
```

Comment:
BTW, assemblySymbol.Identity.Version might not be the most interesting as it doesn't have the full build number or preview suffix.

Informational version might be more useful. (https://docs.microsoft.com/en-us/dotnet/api/system.reflection.assemblyinformationalversionattribute?view=netcore-3.1)

Reply:
Thanks, will test that out.

Reply:
So this is what it looks like `12.5.0-dev.20200712.1+f40a80a7d2fa42155c0d379e2033bdaf656557f6`
It still doesn't include the preview number so I don't think it is an improvement, even though it includes the build date.

Reply:
@weshaggard do you know if there is a way to get the preview version of an assembly at runtime?

Reply:
>@weshaggard do you know if there is a way to get the preview version of an assembly at runtime?

It would be in informational version, it just needs to be build on CI. 

> 12.5.0-dev.20200712.1+f40a80a7d2fa42155c0d379e2033bdaf656557f6

I sort of like it. It includes both date and commit. How bad does it look in UI?

Reply:
pretty bad.. especially if we include it in the dropdown.
I guess we can truncate to first 7 chars of commit.

Reply:
Or we can trim the entire commit SHA. Looks pretty OK that way.

Reply:
The commit SHA seems to be the useful part. Why is the date useful?

Reply:
Ah, I missed your comment about the preview being available only when build through CI. In that case, should we just leave it as is?